crates/nostr/src/event/tag.rs

```diff
@@ -426,6 +427,9 @@ impl From<Tag> for Vec<String> {
                 vec![TagKind::P.to_string(), pk.to_string(), report.to_string()]
             }
             Tag::Reference(r) => vec![TagKind::R.to_string(), r],
+            Tag::RelayMetadata(url, rw) => {
+                vec![TagKind::R.to_string(), url, rw.unwrap_or_default()]
+            }
```

Comment:
Replace this with:

```rust
Tag::RelayMetadata(url, rw) => {
    let mut tag = vec![TagKind::R.to_string(), url];
    if let Some(rw) = rw {
        tag.push(rw);
    }
    tag
}
```

Reply:
Maybe

```rust
Tag::RelayMetadata(url, rw) => {
    vec![Some(TagKind::R.to_string()), Some(url), rw]
        .into_iter()
        .flatten()
        .collect()
}
```

is more elegant?

I'm fine with either way.

Reply:
Yes, it's better!

Reply:
As performance change something?
The first simply push and the second iter all elements (aren't a lot so maybe can be good).

Reply:
Hmm true, first one may be better. I'll stick to your version then.